cloud/passportauth/example/UserRepository.ts

```diff
@@ -22,23 +22,20 @@ const userSeedData = [
 ];
 
 /**
- * A sample user implementation
+ * A sample user service implementation
  */
-export class BaseUser implements User {
-  // Wrap user object
-  constructor(readonly user: any) {
+export class UserService implements User {
```

Comment:
`UserService implements User`
IMHO looks strange here. User is busines object Service is service. 
Mixing this two together is confusing. Let's reword interface.